examples/ae_examples/cvae_dim_example/client.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+from pathlib import Path
+from typing import Sequence, Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+import torchvision.transforms as transforms
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from examples.cvae_dim_example.mnist_model import MnistNet
+from fl4health.clients.basic_client import BasicClient
+from fl4health.preprocessing.autoencoders.vae_dim_reduction import ClientConditionedProcessor
+from fl4health.utils.load_data import load_mnist_data
+from fl4health.utils.metrics import Accuracy, Metric
+from fl4health.utils.random import set_all_random_seeds
+from fl4health.utils.sampler import DirichletLabelBasedSampler
+
+
+class CVAEDimClient(BasicClient):
+    def __init__(self, data_path: Path, metrics: Sequence[Metric], DEVICE: torch.device, condition: str):
+        BasicClient.__init__(self, data_path, metrics, DEVICE)
+        self.condition = condition
+
+    def get_data_loaders(self, config: Config) -> Tuple[DataLoader, DataLoader]:
+        batch_size = self.narrow_config_type(config, "batch_size", int)
+        CVAE_model_path = Path(self.narrow_config_type(config, "CVAE_model_path", str))
```

Comment:
For variable names, typically we don't use capitalized letters for python variables unless its a global constant. So this variable would become `cvae_model_path`

Reply:
There are a few places throughout the PR where this occurs that we can also change 🙂